hollow-jar/web/pom.xml

```diff
@@ -71,6 +100,11 @@
                                     <version>${version.org.wildfly.swarm}</version>
                                     <classifier>hollowswarm</classifier>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.h2database</groupId>
+                                    <artifactId>h2</artifactId>
+                                    <version>1.4.196</version>
```

Comment:
Is there a `version.com.h2database` property that we could use here, instead of hardcoding the version?

Reply:
Actually the property is there and is called `version.com.h2database.h2`.